nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Js create a tag

# Description
Implemented modal for create new tag feature. Only admins are able to post new tags.
Fixes #14 
## Type of change
# Testing Instructions
- Pull down PR branch
- Make sure appsettings.Local.json contains your firebaseProjectId
- Launch API (visual studio) and App (npm start)
- Login with an Admin account
- Navigate to Tags list view
- Click on Create Tag button, modal with tag name input field should appear
- User is able to cancel by clicking on Cancel button, X icon or anywhere outside modal
- If User clicks Save without an empty input field, an alert message should appear, not allowing user to submit
- Once input field is populated and user clicks Save, user should be redirected back to tags list view with the newly created tag
- Double check database for newly created tag
- Login as non-admin user and repeat steps 1-9, you should see an unauthorized message on page and console
# Checklist:
